src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/store/custom.py

```diff
@@ -0,0 +1,257 @@
+                logger.warning('User supplied Key Vault information. For service managed encryption user supplied Key Vault information is ignored.')
+        
+        config.type = encryption_type
+        create_params.encryption_config = config
+        create_params.identity = identity
+    else:
+        create_params.encryption_state = EncryptionState.disabled
+        create_params.identity = None
+        create_params.encryption_config = None
+
+    return client.create(resource_group_name, account_name, create_params)
+
+def update_adls_account(client,
+                        account_name,
+                        tags = None,
+                        resource_group_name = None,
+                        default_group = None,
+                        firewall_state = None,
+                        allow_azure_ips = None,
+                        trusted_id_provider_state = None,
+                        tier = None):
+    if not resource_group_name:
+        resource_group_name = _get_resource_group_by_account_name(client, account_name)
+
+    update_params = DataLakeStoreAccountUpdateParameters()
+
+    if tags:
+        update_params.tags = tags
+
+    if default_group:
+        update_params.default_group = default_group
+
+    if firewall_state:
+        update_params.firewall_state = firewall_state
+
+    if allow_azure_ips:
+        update_params.allow_azure_ips = allow_azure_ips
+
+    if trusted_id_provider_state:
+        update_params.trusted_id_provider_state = trusted_id_provider_state
+
+    if tier:
+        update_params.new_tier = tier
+
+    return client.update(resource_group_name, account_name, update_params)
+
+# firewall customizations
+def add_adls_firewall_rule(client,
+                           account_name,
+                           firewall_rule_name,
+                           start_ip_address,
+                           end_ip_address,
+                           resource_group_name = None):
+    if not resource_group_name:
+        resource_group_name = _get_resource_group_by_account_name(client, account_name)
+
+    create_params = FirewallRule(start_ip_address, end_ip_address)
+    return client.create_or_update(resource_group_name,
+                            account_name,
+                            firewall_rule_name,
+                            create_params)
+
+# filesystem customizations
+def get_adls_item(account_name,
+                  path):
+    return cf_datalake_store_filesystem(account_name).info(path)
+
+def list_adls_items(account_name,
+                    path):
+    return cf_datalake_store_filesystem(account_name).ls(path, detail=True)
+def create_adls_item(account_name,
+                     path,
+                     content=None):
+    client = cf_datalake_store_filesystem(account_name)
+    if content:
+        with client.open(path, mode='wb') as f:
+            f.write(content)
+    else:
+        client.touch(path)
```

Comment:
It's fine if the SDK returns nothing. If you insert the return statement and the SDK returns nothing, then the behavior will just work. If, later on, the SDK changes to return something, then it will automatically pass that through to the user. However, as written, in that scenario the CLI would continue to output nothing in that case, which isn't what we would want. 

I don't recommend returning true on success. Just pass through whatever (if anything) the SDK does and that is perfect. 👍